src/kinematics/src/Shared/Kinematics.lua

```diff
@@ -205,4 +205,4 @@ function Kinematics:_positionVelocity(now)
 	       v0 + a0*dt
 end
 
-return Kinematics
+return Kinematics
```

Comment:
new line here? I mean, this is fine.

Reply:
o what the, oops -- I think my editor automatically trims that when I save